src/encoded/docs/schema-changes.md

```diff
@@ -229,12 +306,12 @@ There are 2 categories when we talk about updating an existing schema:
 
```

Comment:
While I agree that it is possible, I would say here that the whole idea of the upgrade is to prevent such an occurrence.